msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md

```diff
@@ -706,6 +708,80 @@ async def on_teams_team_renamed(
 
 * * *
 
+### Team deleted
```

Comment:
Should we move this to be next to Team restored, to emphasize that the relationship between the two?

Reply:
moved